lib/apk.sh

```diff
@@ -34,6 +34,17 @@ apk_Q() {
   esac
 }
 
+apk_Qe() {
+  packages_marked_to_install="$(cat /etc/apk/world)"
+  installed_packages="$(apk info)"
+
+  while read -r L; do
+    echo "$installed_packages" | grep -Eo "^$L$"
+  done << EOF
+$packages_marked_to_install
```

Comment:
Do think that this works: `apk info | grep --line-regexp -f /etc/apk/world`? I've tested on `apk` tho.

Reply:
(I meant, I haven't tested on apk-based system)

Reply:
It works, but doesn't support the long option name

<details>
  <summary><code>apk info | grep --line-regexp -f /etc/apk/world</code></summary>

```
grep: unrecognized option: line-regexp
BusyBox v1.33.1 () multi-call binary.

Usage: grep [-HhnlLoqvsrRiwFE] [-m N] [-A/B/C N] PATTERN/-e PATTERN.../-f FILE [FILE]...

Search for PATTERN in FILEs (or stdin)

	-H	Add 'filename:' prefix
	-h	Do not add 'filename:' prefix
	-n	Add 'line_no:' prefix
	-l	Show only names of files that match
	-L	Show only names of files that don't match
	-c	Show only count of matching lines
	-o	Show only the matching part of line
	-q	Quiet. Return 0 if PATTERN is found, 1 otherwise
	-v	Select non-matching lines
	-s	Suppress open and read errors
	-r	Recurse
	-R	Recurse and dereference symlinks
	-i	Ignore case
	-w	Match whole words only
	-x	Match whole lines only
	-F	PATTERN is a literal (not regexp)
	-E	PATTERN is an extended regexp
	-m N	Match up to N times per file
	-A N	Print N lines of trailing context
	-B N	Print N lines of leading context
	-C N	Same as '-A N -B N'
	-e PTRN	Pattern to match
	-f FILE	Read pattern from file
```

</details>

Reply:
Great. The short option is just fine and supported by both `busybox` and `bash` (https://github.com/mondeja/pacapt/runs/2993122835?check_suite_focus=true)